helper/tlsutil/config.go

```diff
@@ -60,6 +62,9 @@ type Config struct {
 	// KeyFile is used to provide a TLS key that is used for serving TLS connections.
 	// Must be provided to serve TLS connections.
 	KeyFile string
+
+	// Utility to dynamically reload TLS configuration.
```

Comment:
Go's style is to put the name of the var/func as the first word in the description so that searching for terms jumps to their comment first (and some tools even take advantage of this idiom!):

```
// KeyLoader dynamically reloads TLS configuration
```

or similar.